rsciio/edax/_api.py

```diff
@@ -952,29 +950,50 @@ def spd_reader(
     ]
 
 
-def file_reader(filename, record_by="spectrum", endianess="<", **kwargs):
+spd_reader.__doc__ %= (FILENAME_DOC, LAZY_DOC, ENDIANESS_DOC, RETURNS_DOC)
+
+
+def file_reader(filename, lazy=False, endianess="<", **kwargs):
     """
+    Reads ``.spc`` (spectrum) or ``.spd`` (spectrum image) files from EDAX
+    Genesis or TEAMS software.
 
     Parameters
     ----------
-    filename : str
-        Name of file to read
-    record_by : str
-        EDAX EDS data is always recorded by 'spectrum', so this parameter
-        is not used
-    endianess : char
-        Byte-order of data to read
+    %s
+    %s
+    %s
+    load_all_spc : bool, Default=False
+        Switch to control whether the complete .spc header is read, or just the
+        important parts for import into HyperSpy.
+    spc_fname : None or str
+        For spd files only.
+        Name of file from which to read the spectral calibration. If data
+        was exported fully from EDAX TEAM software, an .spc file with the
+        same name as the .spd should be present.
+        If `None`, the default filename will be searched for.
+        Otherwise, the name of the .spc file to use for calibration can
+        be explicitly given as a string.
+    ipr_fname : None or str
+        For spd files only.
+        Name of file from which to read the spatial calibration. If data
+        was exported fully from EDAX TEAM software, an .ipr file with the
+        same name as the .spd (plus a "_Img" suffix) should be present.
+        If `None`, the default filename will be searched for.
+        Otherwise, the name of the .ipr file to use for spatial calibration
+        can be explicitly given as a string.
     **kwargs
-        Additional keyword arguments supplied to the readers
-
-    Returns
-    -------
+        Remaining arguments are passed to the Numpy ``memmap`` function.
 
+    %s
```

Comment:
To give more visibility to available resources:

```suggestion
    %s

    Notes
    -----
    The file specification is available at :ref:`edax-file_specification`.
```